plugins/completion/cmp/sources/_mk-cmp-plugin.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+{
+  name ? pluginName,
+  pluginName,
+  sourceName,
+  defaultPackage ? pkgs.vimPlugins.${pluginName},
+  maintainers ? [ lib.maintainers.GaetanLepage ],
+  imports ? [ ],
+  ...
+}@args:
+helpers.vim-plugin.mkVimPlugin config (
```

Comment:
Not sure if this is best using the upcoming `mkPlugin`, or perhaps using `mkNeovimPlugin` but with `callSetup ? false` and `hasSettings ? false` as defaults

Reply:
I propose to revisit this once the refactor is done no ?

Reply:
Sure